docs/reference/api/connectivity/bluetooth/GattClient.md

```diff
@@ -1,6 +1,26 @@
 ## GattClient
 
-[Add description here.]
+GattClient defines procedures required for interacting with a remote GattServer.
+
+#### Discovery procedures
+
+A GattServer hosts a fixed set of services. These services are a logical composition of characteristics that may be discovered, read, written or also broadcast their state to a connected client. These characteristics may also contain metainformation named characteristic descriptors. A characteristic descriptor may be used to indicate the unit used for a characteristic value, describe in a textual form the characteristic purpose or allow a client to register for notification of updates of the characteristic value.
```

Comment:
The switch from the passive to active voice in the 2nd sentence is confusing ("may be discovered, read, written" (passive) and "broadcast their state" (active).) Maybe break up the tenses with a conjunction: "These services are a logical composition of characteristics that may be discovered, read or written, and can broadcast their state to a connect client."

Conunction missing between 'metainformation' and 'named characteristic descriptors.' (3rd sentence)

"Used" is repetitive in the beginning of the 4th sentence